modules/administration/pages/content-lifecycle.adoc

```diff
@@ -113,6 +113,46 @@ If there is no previous environment, it is left empty until the project sources
 
 
 
+[#filter-templates]
+== Filter Templates
+
+To help with creating filters for some common scenarios, {productname} offers filter templates.
+When applied, these templates help create a set of pre-defined filters, tailored for a specific use case.
+
+This section describes available templates and their usages.
+
+
+
+=== Live patching based on a SUSE product
+
+In a project that contains live patching, regular kernel packages must be excluded so that only live patch packages are offered as updates to clients. On the other hand, already installed regular kernel packages must still be included to keep system integrity.
+
+When applied, this template creates two filters required to achieve this behavior:
+
+* Deny patches that contain `kernel-default` package greater than a specific kernel version
+* Deny patches that contain `reboot_suggested` keyword
+
+[IMPORTANT]
+====
+The second "catch-all" filter is a safeguard to make sure the assigned clients are never required a restart in the future. You can choose not to attach it to the project if this behavior is not desired as it might block other important updates.
```

Comment:
We have this strange policy (I think it help adoc editors): New sentence = new line
```suggestion
The second "catch-all" filter is a safeguard to make sure the assigned clients are never requiring a restart in the future.
You can choose not to attach it to the project if this behavior is not desired because it might block other important updates.
```